public/app/features/dashboard-scene/saving/SaveDashboardAsForm.tsx

```diff
@@ -86,8 +89,19 @@ export function SaveDashboardAsForm({ dashboard, changeInfo }: Props) {
         </Stack>
       </>
     );
+  };
+  function renderFooter(error?: Error) {
+    if (isNameExistsError(error)) {
+      return <NameAlreadyExistsError cancelButton={cancelButton} saveButton={saveButton} />;
+    }
+    return normalFooter(error);
```

Comment:
could probably revert it to how `renderFooter` used to be and just have 
```
  function renderFooter(error?: Error) {
    const = formValuesMatchContentSent = formValues.title.trim() === contentSent.title && formValues.folder.uid === contentSent.folderUid;
    if (isNameExistsError(error) && formValuesMatchContentSent) {
      return <NameAlreadyExistsError cancelButton={cancelButton} saveButton={saveButton} />;
    }
    
    return (...
```
then you wouldn't need the `isRefreshed` state variable or the `useEffect`